terraform/README.md

```diff
@@ -0,0 +1,63 @@
+# Terraform module for kubeflow-dashboard
+
+This is a Terraform module facilitating the deployment of the kubeflow-dashboard charm, using the [Terraform juju provider](https://github.com/juju/terraform-provider-juju/). For more information, refer to the provider [documentation](https://registry.terraform.io/providers/juju/juju/latest/docs). 
+
+## Compatibility
+This terraform module is compatible with charms of version >= 1.7 due to changes in the charm's relations.
```

Comment:
The new `logging` relation was added to 1.9, so there's an incompatibility with lower versions, right?
```suggestion
This terraform module is compatible with charms of version >= 1.9 due to changes in the charm's relations.
```